src/main/java/com/spotinst/sdkjava/model/DefaultStaticTargetGroups.java

```diff
@@ -0,0 +1,98 @@
+package com.spotinst.sdkjava.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Created by aharontwizer on 8/24/15.
+ */
+public class DefaultStaticTargetGroups {
+
+    //region Members
+    @JsonIgnore
```

Comment:
@JsonIgnoreProperties(ignoreUnknown = true)
@JsonInclude(JsonInclude.Include.NON_NULL)

above 2 lines are required in all BL classes as well